code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1035,6 +1029,11 @@
 		for(var/obj/item/tool/lighter/Z in contents)
 			if(Z.turn_off(src))
 				goes_out++
+	if(sparks & light_off)
+		var/datum/effect_system/spark_spread/spark_system = new
+		spark_system.set_up(5, 0, src)
+		spark_system.attach(src)
+		spark_system.start(src)
```

Comment:
shouldnt this just parentcall?

Reply:
No, or when xenos are slashing multiple sparks would be created, wich is useless